guide/daily-spending-wallet/send-fees.md

```diff
@@ -0,0 +1,205 @@
+%}
+
+# Send fees
+{:.no_toc}
+
+Fees are a necessity in a market-based network, both the base layer and lightning. They can vary drastically based on which layer the transaction is being made on, as well as the current network state, transaction complexity, whether any additional services were used, and more. To simplify this complexity for users, it is important to provide clear and actionable information and options.
+
+## Lightning fees
+
+Let's take a look at how fees surface in a regular send flow, and how users can quickly pull up additional details.
+
+{% include image-gallery.html pages = page.images_breakdown %}
+
+#### Routing fees
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lightning-fee-sample.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lightning-fee-sample@2x.png"
+   alt-text = "How to display a lightning routing fee"
+   width = 400
+   height = 85
+   layout = "float-right-desktop"
+%}
+
+On the lightning network, payments are passed between nodes to get from the sender to the receiver. Each of those nodes may charge a base fee and a second fee based on a percentage of the amount forwarded. Fees paid can vary, but are typically in the single-digit or double-digit Satoshi range (a small fraction of on-chain fees).
+
+</div>
+
+#### Service fees
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lsp-fee-sample.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lsp-fee-sample@2x.png"
+   alt-text = "How to display a lightning service fee"
+   width = 400
+   height = 85
+   layout = "float-right-desktop"
+%}
+
+Lightning wallets may require [additional services]({{ '/guide/how-it-works/lightning-services/' | relative_url }}) to solve certain usability issues. An example being a lightning service provider (LSP) opening payment channels and providing [inbound liquidity]({{ '/guide/how-it-works/liquidity/' | relative_url }}) for the user, so they can receive payments. As these services are offered by third parties, additional fees may be charged.
+
+</div>
+
+## On-chain fees
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/onchain-fee-sample.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/onchain-fee-sample@2x.png"
+   alt-text = "How to display an onchain transaction fee"
+   width = 400
+   height = 252
+   layout = "float-right-desktop"
+%}
+
+This fee is largely dependent on how many other transactions are currently [waiting]({{ '/guide/glossary/#mempool' | relative_url }}) to be processed on the base layer as a whole. The [average fee](https://ycharts.com/indicators/bitcoin_average_transaction_fee) in January 2021 was $0.63, and $28.60 in April 2021.
```

Comment:
I really like the fee comparison here. It helps to understand how wide the range can be.